### PR TITLE
Make width and height prop values out of scale

### PR DESCRIPTION
**Pull request by vladmoroz**
The `width` and `height` props won't map to space scale anymore. This is because in the vast majority of cases, width and height were not set to space scale, and with that, space scale as an IDE autocomplete suggestion felt odd/misleading.

### CODE REVIEW

File: packages/radix-ui-themes/changelog.md [hunk 1]
```diff
@@ -24,6 +24,8 @@
     - `flexBasis`, `flexShrink`, `flexGrow`
     - `gridColumn`, `gridColumnStart`, `gridColumnEnd`
     - `gridRow`, `gridRowStart`, `gridRowEnd`
+  - **[Breaking]** The `width` and `height` props don't map to space scale anymore. This is because in the vast majority of cases, width and height were not set to space scale, and with that, space scale as an IDE autocomplete suggestion felt odd/misleading.
+    - **[Upgrade guide]** Find and replace your `width` and `height` prop usage with the corresponding [space scale](https://github.com/radix-ui/themes/blob/main/packages/radix-ui-themes/src/styles/tokens/space.css) steps, e.g. `width="9"` would need to be changed to `width="64px"`, and so on for other steps.
```
Comment (benoitgrelard) [inline packages/radix-ui-themes/changelog.md:28]: Should we suggest they could also refer to the CSS variables?

File: packages/radix-ui-themes/src/components/container.tsx [hunk 2]
```diff
@@ -2,8 +2,14 @@ import * as React from 'react';
 import classNames from 'classnames';
 import { Slot } from '@radix-ui/react-slot';
 import { containerPropDefs } from './container.props.js';
-import { extractProps, getRoot } from '../helpers/index.js';
-import { deprecatedLayoutPropDefs, layoutPropDefs, marginPropDefs } from '../props/index.js';
+import { extractProps, getRoot, subtract } from '../helpers/index.js';
+import {
+  deprecatedLayoutPropDefs,
```
Comment (benoitgrelard) [inline packages/radix-ui-themes/src/components/container.tsx:7]: Just checking, are we going to remove the deprecated stuff now that we're making a major version?

File: packages/radix-ui-themes/src/components/container.tsx [hunk 3]
```diff
@@ -15,36 +21,46 @@ interface ContainerProps
...
+  ({ width, minWidth, maxWidth, ...props }, forwardedRef) => {
+    const {
+      asChild,
+      children: childrenProp,
+      className,
+      ...containerProps
+    } = extractProps(
+      props,
+      containerPropDefs,
+      subtract(layoutPropDefs, widthPropDefs),
```
Comment (benoitgrelard) [inline packages/radix-ui-themes/src/components/container.tsx:34]: why do we need to do this?

File: packages/radix-ui-themes/src/styles/utilities/height.css
Comment (benoitgrelard) [inline packages/radix-ui-themes/src/styles/utilities/height.css:1]: we need to remove those imports somewhere probably right?

Comment (vladmoroz) [reply packages/radix-ui-themes/src/styles/utilities/height.css:1]: Imports? This file still has contents, I just removed the hardcoded space scale values CSS

File: packages/radix-ui-themes/changelog.md (hunk 1 above)
Comment (vladmoroz) [reply packages/radix-ui-themes/changelog.md:28]: Good point, will add!

File: packages/radix-ui-themes/src/components/container.tsx (hunk 2 above)
Comment (vladmoroz) [reply packages/radix-ui-themes/src/components/container.tsx:7]: Yep, just separately

File: packages/radix-ui-themes/src/components/container.tsx (hunk 3 above)
Comment (vladmoroz) [reply packages/radix-ui-themes/src/components/container.tsx:34]: I guess we don't need to with width props having no default right now, but we'd need to if these prop defs were setting a default value. Should we remove and use this only as needed?

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**